### ISSUE TITLE
Question re logic applied to queried records

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1363]
This arises from this record:
https://irecord.org.uk/record-details?occurrence_id=14009276

The record was marked as "Accepted" two years ago, but a comment was subsequently added that cast doubt on it. Ten months ago, the record was queried, and it looks like it was queried three times (twice by email and once via a comment). However, the record is still showing as "Accepted" and has been included in the data we send to NBN Atlas.

So it appears that adding a query to a record that was previously accepted does not change its verification status.

I would prefer it if querying a record also resulted in that record being set back to the "pending" state. Is there any reason that we can't do that?

It seems wrong that we are sharing "Accepted" records, e.g. to NBN Atlas, when in fact some of them have outstanding queries.

**Comment by johnvanbreda** [comment]
Just to confirm - the requirement here is to reset the verification status to pending if a query is added?

**Comment by kitenetter** [comment]
@johnvanbreda yes, that feels like the correct behaviour to me, but let me know if you can see any problem with changing to that requirement.

And thinking about it, I would also suggest that if a queried record is subsequently given a new verification status, it shuld no longer be flagged as queried.

In other words, a queried record is by definition "unconfirmed", and a verified record is by definition no longer subject to query (although of course it could be reverified if further evidence is received).

**Comment by kitenetter** [comment]
@johnvanbreda did the above logic for queried records get applied?
